app/services/events/validate_creation_service.rb

```diff
@@ -53,32 +54,20 @@ def validate_create_batch
     end
 
     def validate_create
-      return invalid_customer_error unless customer
-      return missing_subscription_error unless valid_subscription?
+      return invalid_customer_error if params[:external_customer_id] && !customer
+      return missing_subscription_error if params[:external_subscription_id].blank? && subscriptions.count > 1
+      return missing_subscription_error if subscriptions.empty?
+      if params[:external_subscription_id] && !subscriptions.pluck(:external_id).include?(params[:external_subscription_id])
```

Comment:
Layout/LineLength: Line is too long. [124/120] (https://rubystyle.guide#max-line-length)